lib/spring/server.rb

```diff
@@ -46,6 +51,7 @@ def boot
     def start_server
       server = UNIXServer.open(env.socket_name)
       log "started on #{env.socket_name}"
+      eager_fork! if eager_fork?
```

Comment:
Should we add somth like `…if eager_fork?|| !env.JRuby? ` ?
JRuby didn't implement fork.

Reply:
It's actually calling `ApplicationManager#restart` to start preloading
https://github.com/rails/spring/blob/v2.0.2/lib/spring/application_manager.rb#L29

Maybe it can be renamed to eager_preload